docs/deployment/apk-generation-for-dev.md

```diff
@@ -12,12 +12,16 @@
     - MYAPP_UPLOAD_KEY_ALIAS=xxx
     - MYAPP_UPLOAD_STORE_PASSWORD=xxx
     - MYAPP_UPLOAD_KEY_PASSWORD=xxx
-7. Change the App Version Number in `android/app/build.gradle` (in `defaultConfig.versionCode/versionName`)
+7. Increment the App Version Number in `android/app/build.gradle` (in `defaultConfig.versionCode/versionName`)
 
     ![android/app/build.gradle](../assets/images/defaultConfigVersion.jpg)
 
-8. Generate the APK or AAB:
-   - To generate an APK: In the directory android, run the command `./gradlew assembleRelease`
-   - To generate an AAB: In the directory android, run the command `./gradlew bundleRelease`
-9. The release will be in `android/app/build/outputs/(apk|bundle)/release/(app-release.apk|app.aab)`
-10. Rename the release to 'DrCannabis-<VERSION_NAME>-YYYYMMDDHHMM.(apk|aab)'
+8. Make sure you have JDK 8 OpenJDK installed. (jdk8-openjdk for Arch linux)
+
+9. Generate the APK or AAB:
+   - To generate an APK, run `npm run generate:apk`
+   - To generate an AAB, run `npm run generate:aab`
+
+10. The release will be in `release/DrCannabis-<timestamp>.(apl|aab)`
```

Comment:
- **Typo:** `apl` should be `apk`
- **Tip:** in MarkDown, if you write lists with the same number, it's rendered sequentially. Example:

**If I write:**
```markdown
1. First item
1. Second item
1. Third item
```
**The output is:**
1. First item
1. Second item
1. Third item